tests/unit/attestations/test_services.py

```diff
@@ -80,37 +79,18 @@ class TestIntegrityService:
     def test_interface_matches(self):
         assert verifyClass(IIntegrityService, IntegrityService)
 
-    def test_create_service(self):
-        request = pretend.stub(
-            find_service=pretend.call_recorder(
-                lambda svc, context=None, name=None: None
-            ),
-        )
-
-        assert IntegrityService.create_service(None, request) is not None
-        assert not set(request.find_service.calls) ^ {
-            pretend.call(IFileStorage, name="archive"),
-            pretend.call(IMetricsService),
-        }
-
-    def test_persist_attestations(self, db_request, monkeypatch):
-        @pretend.call_recorder
-        def storage_service_store(path: str, file_path, *_args, **_kwargs):
-            expected = VALID_ATTESTATION.model_dump_json().encode("utf-8")
-            with open(file_path, "rb") as fp:
-                assert fp.read() == expected
-
-            assert path.endswith(".attestation")
+    def test_create_service(self, db_request):
+        service = IntegrityService.create_service(None, db_request)
```

Comment:
FLAG: I've changed this test to reuse the `db_request` fixture.

The newly created service gets its two dependent services directly from the fixture. 
I wanted to test that they were correctly created but the `service.storage.base.exists` seems a bit off.

Reply:
> I wanted to test that they were correctly created but the service.storage.base.exists seems a bit off.

Yeah, let's not assert that here -- that's an implementation detail from an unrelated interface/service.